mindee/parsing/common/document.py

```diff
@@ -57,3 +58,21 @@ def __str__(self) -> str:
             f":Filename: {self.filename}\n\n"
             f"{self.inference}"
         )
+
+    def _inject_full_text_ocr(self, raw_prediction: StringDict):
+        if len(raw_prediction["inference"]["pages"]) < 1:
+            return
+        if "extras" not in raw_prediction["inference"]["pages"][0]:
+            return
+        if "full_text_ocr" not in raw_prediction["inference"]["pages"][0]["extras"]:
+            return
+        artificial_text_obj = {"content": ""}
+        artificial_text_obj["content"] += "\n".join(
+            page["extras"]["full_text_ocr"]["content"]
+            for page in raw_prediction["inference"]["pages"]
+        )
+        if not hasattr(self, "extras"):
+            self.extras = Extras({"full_text_ocr": artificial_text_obj})
+        else:
+            assert self.extras is not None
```

Comment:
what's the point of raising an exception here?

Reply:
Guess I'm missing something, which part?

Reply:
Ah, that part. It was there so that mypy wouldn't complain about the possibly None value. Which I believe was a mistake in the first place. Fixed.